README.md

```diff
@@ -1,3 +1,17 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+#Database instructions:
+
+Creating the databases
+We'll have two databases in this project:
+one for real-looking dev data, and another for simpler test data.
+
+You will need to create two .env files for your project:
+.env.test and .env.development.
+
+Into each, add PGDATABASE=, with the correct database name for that environment (see /db/setup.sql for the database names).
+Double check that these .env files are .gitignored.
```

Comment:
These instructions are looking great. Remember though, we can afford to be EXTRA explicit. These instruction will likely be read by someone who has no express experience or pg experience. So tell them exactly what to write in exactly which file.